common/convert/convert.go

```diff
@@ -81,3 +84,83 @@ func BoolPtr(condition bool) *bool {
 	b := condition
 	return &b
 }
+
+// IntToCommaSeparatedString converts an int to a comma separated string at the thousand point
+// eg 1000 becomes 1,000
+func IntToCommaSeparatedString(number int64, thousandsSep string) string {
+	neg := false
+	if number < 0 {
+		number = -number
+		neg = true
+	}
+	str := fmt.Sprintf("%v", number)
+	return numberToCommaSeparatedString(str, 0, "", thousandsSep, neg)
+}
+
+// FloatToCommaSeparatedString converts a float to a comma separated string at the thousand point
+// eg 1000 becomes 1,000
+func FloatToCommaSeparatedString(number float64, decimals uint, decPoint, thousandsSep string) string {
```

Comment:
This func name is technically incorrect as a user can supply whatever they want for the `thousandsSep` param. I think a better name would be "FloatToHumanFriendlyString" or you can drop the thousandsSep param and make it always a comma as the name suggests